manifests/globals.pp

```diff
@@ -77,6 +77,7 @@
         }
       } else {
         case $globals_php_version {
+          /^5\.6/,
```

Comment:
where was this before?

Reply:
yupp, that's what travis is showing https://travis-ci.org/voxpupuli/puppet-php/jobs/391234923#L1338

Reply:
so what should We Do? Drop Debian 7 Support or more code?

Reply:
Everything that is Debian 7 specific can go away. We don't support it anymore. But we still have to support PHP 5.6.

Reply:
actually, we could drop PHP 5.6 now.